package.json

```diff
@@ -173,7 +173,8 @@
     "portsAttributes",
     "findTextInFiles",
     "workspaceTrust",
-    "resolvers"
+    "resolvers",
```

Comment:
Once all extensions that need to be switched over to `"tunnels"` are switched, I will update the checkes in VS Code and we can remove `"resolvers"`.